data/pathfinder_pfs/paizo/adventure_path/curse_of_the_crimson_throne/pfs_players_guide/pfs_curse_of_the_crimson_throne_players_guide.pcc

```diff
@@ -40,5 +40,5 @@ COPYRIGHT:PCGen dataset conversion for "Curse of the Crimson Throne Player's Gui
 # Equipment: all equipment on pages 12-13 (except hide shirt); Feats: Acadamae Graduate, Crossbow Mastery
 EQUIPMENT:@/pathfinder/paizo/adventure_path/curse_of_the_crimson_throne/players_guide/cotctpg_equip_arms_armor.lst|(EXCLUDE:Hide Shirt)
 WEAPONPROF:@/pathfinder/paizo/adventure_path/curse_of_the_crimson_throne/players_guide/cotctpg_profs_weapon.lst
-ABILITY:@/pathfinder/paizo/adventure_path/curse_of_the_crimson_throne/players_guide/cotctpg_feats.lst|(INCLUDE:CATEGORY=Special Ability,Acadamae Graduate)
+FEAT:@/pathfinder/paizo/adventure_path/curse_of_the_crimson_throne/players_guide/cotctpg_feats.lst|(INCLUDE:CATEGORY=Special Ability,Acadamae Graduate)
```

Comment:
It wasn’t changed back to feat. I started this project a week before FEAT was decided to be removed. And the line changed for other reasons.

 

From: Andrew Maitland [mailto:notifications@github.com] 
Sent: Monday, January 26, 2015 12:30 AM
To: PCGen/pcgen
Cc: mjmeans
Subject: Re: [pcgen] DATA-2327 - PFS updates (#132)

 

In data/pathfinder_pfs/paizo/adventure_path/curse_of_the_crimson_throne/pfs_players_guide/pfs_curse_of_the_crimson_throne_players_guide.pcc <https://github.com/PCGen/pcgen/pull/132#discussion_r23515107> :

> @@ -40,5 +40,5 @@ COPYRIGHT:PCGen dataset conversion for "Curse of the Crimson Throne Player's Gui
>  # Equipment: all equipment on pages 12-13 (except hide shirt); Feats: Acadamae Graduate, Crossbow Mastery
>  EQUIPMENT:@/pathfinder/paizo/adventure_path/curse_of_the_crimson_throne/players_guide/cotctpg_equip_arms_armor.lst|(EXCLUDE:Hide Shirt)
>  WEAPONPROF:@/pathfinder/paizo/adventure_path/curse_of_the_crimson_throne/players_guide/cotctpg_profs_weapon.lst
> -ABILITY:@/pathfinder/paizo/adventure_path/curse_of_the_crimson_throne/players_guide/cotctpg_feats.lst|(INCLUDE:CATEGORY=Special Ability,Acadamae Graduate)
> +FEAT:@/pathfinder/paizo/adventure_path/curse_of_the_crimson_throne/players_guide/cotctpg_feats.lst|(INCLUDE:CATEGORY=Special Ability,Acadamae Graduate)

Why was this changed back to feat?

—
Reply to this email directly or view it on GitHub <https://github.com/PCGen/pcgen/pull/132/files#r23515107> .  <https://github.com/notifications/beacon/ADyapw3f2Ni9Xk0UjKYj8YT5tvW3C9A7ks5nleRqgaJpZM4DXDYk.gif>